.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
When config is short like this, I prefer to use the package.json version. Applies to husky too

Reply:
Will revert that